skcosmo/selection/FPS.py

```diff
@@ -51,16 +53,25 @@ class _BaseFPS:
                  is random
     :type idxs: list of int, None
 
+    :param progress_bar: option to use `tqdm <https://tqdm.github.io/>`_
+                         progress bar to monitor selections
+    :type progress_bar: boolean
+
     """
 
-    def __init__(self, tol=1e-12, idxs=None):
+    def __init__(self, tol=1e-12, idxs=None, progress_bar=False):
         if idxs is not None:
             self.idx = idxs
         else:
             self.idx = [np.random.randint(self.product.shape[0])]
         self.distances = np.min([self.calc_distance(i) for i in self.idx], axis=0)
         self.distance_selected = np.nan * np.zeros(self.product.shape[0])
 
+        if progress_bar:
+            self.progress_report = get_progress_bar()
```

Comment:
I know this was my suggestion, but I find `report_progress` to be a better name here

Sorry for the back & forth here =/

Reply:
Any particular reason?

Reply:
I usually prefer naming function using `<action verb>_<modifiers>` e.g. `compute_distances` instead of `distances_compute`. I find it to flow better when reading the code